package.json

```diff
@@ -82,6 +82,7 @@
   },
   "dependencies": {
     "colors": "^1.4.0",
+    "express": "^4.17.1",
```

Comment:
First thought before anything else: overkill for a static server! 😆

Reply:
Yeah, was thinking the same... at the same time, it's nice to use. Do you have a suggestion for a good alternative. Generally, I use https://www.npmjs.com/package/http-server but it doesn't look like it has an API.

Reply:
I used `serve-handler` in spec-prod